administration/console.md

```diff
@@ -16,23 +16,29 @@ The console offers the option to:
 ## Accessing the Console
 
 Accessing the console depends on the start mode of openHAB.
+
+### Started with Script
 The manually start using shell/batch script ends directly in the console.
 
-If openHAB runs a service, the console can be accessed using ssh to the openHAB host on port 8101.
+### Running as Service
+If openHAB runs a service, the console can be accessed using ssh from the openHAB host on localhost:8101. It is intentionally not available from remote hosts due to security concerns.
```

Comment:
Minor nitpick, could you please use one line per sentence? This helps avoiding git history churn 😄